sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/additonalNpmConfig/project/build.properties

```diff
@@ -1 +0,0 @@
-sbt.version=0.13.15
```

Comment:
Why remove that? I think keeping it (and setting it to `1.0.1`) makes it possible to load the project whatever sbt version is installed.

Reply:
If we keep it, it will force the test to only work with that specific sbt version. Most tests I assume we want to run for both versions of sbt. Those we want to fix to a specific sbt version will have to be configured so the CI can filter them out. I've already added a mechanism to do this as pointed out elsewhere.

Reply:
OK I see! I didn’t notice that the tests were executed with both sbt 1.0 and 0.13. This is great actually :) We should not have to introduce a mechanism testing whether a test should be run with sbt 1.0 or 0.13 only.

Reply:
We need that mechanism for at least `sbt-web-scalajs-bundler/play` until the Play sbt plugin has been published for sbt 1.0 (https://github.com/playframework/playframework/issues/7261). However, based on @sjrd's concern, we should probably also add tests that target the minimum version of sbt 0.13 and sbt 1.0 that scalajs-bundler supports to ensure documentation about those requirements stays up to date.